src/GitHub.VisualStudio/UI/GitHubPane.cs

```diff
@@ -131,6 +144,16 @@ public override void OnToolWindowCreated()
             UpdateSearchHost(pane?.IsSearchEnabled ?? false, pane?.SearchQuery);
         }
 
+        void ShowInitializing()
+        {
+            View = new Label { Content = "Initializing MEF" };
```

Comment:
I know you were asking what to display here. We probably shouldn't display a spinner as it will be _really_ jerky while MEF is initializing. I think the following should suffice:

```
View = new TextBlock
{
	Text = "Initializing...",
	HorizontalAlignment = HorizontalAlignment.Center,
	VerticalAlignment = VerticalAlignment.Center,
};
```

I changed the text to just "Initializing..." as MEF might not mean much to users.